src/Compilers/CSharp/Test/Semantic/Semantics/AmbiguousOverrideTests.cs

```diff
@@ -188,15 +185,9 @@ public override void Method(int a, long b)
                 // (12,26): error CS0462: The inherited members 'Base<T, U>.Method(T)' and 'Base<T, U>.Method(int)' have the same signature in type 'Derived', so they cannot be overridden
                 //     public override void Method(int a)
                 Diagnostic(ErrorCode.ERR_AmbigOverride, "Method").WithArguments("Base<T, U>.Method(T)", "Base<T, U>.Method(int)", "Derived"),
-                // (4,25): warning CS1957: Member 'Derived.Method(int)' overrides 'Base<int, int>.Method(int)'. There are multiple override candidates at run-time. It is implementation dependent which method will be called.
-                //     public virtual void Method(T x) { }
-                Diagnostic(ErrorCode.WRN_MultipleRuntimeOverrideMatches, "Method").WithArguments("Base<int, int>.Method(int)", "Derived.Method(int)"),
```

Comment:
>Diagnostic(ErrorCode.WRN_MultipleRuntimeOverrideMatches, "Method").WithArguments("Base<int, int>.Method(int)", "Derived.Method(int)"), [](start = 16, length = 134)

Same comment as for other tests affected by the removal of WRN_MultipleRuntimeOverrideMatches

Reply:
There is no runtime behavior for a program that does not compile.

---
In reply to: [442519006](https://github.com/dotnet/roslyn/pull/44025#discussion_r442519006) [](ancestors = 442519006)